documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -59,6 +59,8 @@ include::modules/compatibility-guidelines.adoc[leveloffset=+2]
 
 You can install the {operator-name} by using the {ocp} web console or the command line interface (CLI).
 
+In {project-first} version 2.4 and later, the {operator-name} includes the {project-short} web console.
```

Comment:
```suggestion
In {project-first} version 2.4 and later, the {operator-name} includes the {project-short} plugin for Openshift web console.
```

Reply:
forklift web console is not neccarily the same as forklift plugin for openshift web console, I think we need to be more clear here